### PR TITLE
Partner highlight for SageMaker, Haystack, and OpenSearch

### PR DESCRIPTION
**Pull request by dtaivpp**
### Description
Adds a partner highlight blog for the "Build production-ready generative AI applications for enterprise search using Haystack pipelines and Amazon SageMaker JumpStart with LLMs"
 
### Issues Resolved
N/A

### Check List
- [X] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: _posts/2023-08-25-sagemaker-haystack-opensearch.markdown [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+  # The type of the twitter card. The value of summary_large_image is used if there is an image defined in the front matter. Otherwise the value of summary is used. However, an explicit type can be defined here for example if a player type is needed for a video.
+  type: summary_large_image
+
+---
+
+<img src="/assets/media/blog-images/2023-08-25-sagemaker-haystack-opensearch/Architecture.png
+" alt="Haystack, Sagemaker, and OpenSearch Architecture for Retreval Augmented Generation"/>{: .img-fluid}
+
+Ever wanted to get started with Generative AI but didn't know where to start? Check out AWS Machine Learning blog on, "Building a production-ready generative AI applications for enterprise search using Haystack pipelines and Amazon SageMaker JumpStart with LLMs". In this article we discuss how retrieval agmented generation (RAG) works and how to set up a RAG pipeline with Haystack, OpenSearch, and SageMaker Jumpstart.
```
Comment (natebower) [inline _posts/2023-08-25-sagemaker-haystack-opensearch.markdown:49]:
```suggestion
Ever wanted to get started with generative AI but didn't know where to start? Check out the AWS Machine Learning blog post "Build production-ready generative AI applications for enterprise search using Haystack pipelines and Amazon SageMaker JumpStart with LLMs". In this article we discuss how retrieval augmented generation (RAG) works and how to set up a RAG pipeline with Haystack, OpenSearch, and SageMaker JumpStart.
```

File: _posts/2023-08-25-sagemaker-haystack-opensearch.markdown [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    - dtaivpp
+    - tuanacelik
+
+category:
+  - partner-highlight
+
+date: 2023-08-25 00:00:01 -0700
+
+# An excerpt of the blog post. It is used in the blog post list view, and in the home page what's new list of N most recent blog posts. It is also used as a fallback value for the twittercard:description field if not explictly defined in the front matter.
+excerpt: "Here we will show how to build an end-to-end generative AI application for enterprise search with Retrieval Augmented Generation (RAG) using Haystack, OpenSearch, and Sagemaker Jumpstart."
```
Comment (natebower) [inline _posts/2023-08-25-sagemaker-haystack-opensearch.markdown:12]:
```suggestion
excerpt: "Here we will show how to build an end-to-end generative AI application for enterprise search with Retrieval Augmented Generation (RAG) using Haystack, OpenSearch, and Sagemaker JumpStart."
```

**Review by natebower — APPROVED** [review]
@dtaivpp Please see my suggested changes and let me know if you have any questions. Thanks!

**Review by krisfreedain — APPROVED** [review]
LGTM!

**Pull request merged**